### ISSUE TITLE
How to reproduce the results of the paper?

### ISSUE THREAD

**Issue by kaisugi** [TeamLab/bert-gcn-for-paper-citation#7]
Hi.
I tried to replicate the results of your paper, but for some reason the numbers I got are different from what is in the paper.

Specifically I wish to get the score of BERT w/o GCN model (in the red-bordered areas of the figure below).

![68747470733a2f2f692e696d6775722e636f6d2f6f6464534341482e706e67](https://user-images.githubusercontent.com/36184621/95820789-be444280-0d63-11eb-9446-e98f308ded51.png)

What I tried to do was as follows:

1. clone this repository and install the dependencies required
2. download the **BERT-base** model from https://github.com/google-research/bert and put the files into `./pre_train/bert_base_uncased` folder
3. download the `Full Context PeerRead` csv dataset described in README, and set it in `./glue/ACRS` folder
4. run the script below

```
  ## PeerRead : python run_classifier.py --model=bert_base --dataset=PeerRead --do_train=true --do_predict=true \
  --data_dir=./glue/ACRS --vocab_file=./pre_train/bert_base_uncased/vocab.txt \
  --bert_config_file=./pre_train/bert_base_uncased/bert_config.json \
  --init_checkpoint=./pre_train/bert_base_uncased/bert_model.ckpt \
  --max_seq_length=50 --train_batch_size=16 --learning_rate=2e-5 \
  --num_train_epochs=30.0 --output_dir=./output --frequency=5
```

<br>

Am I missing any procedures?

**Comment by eshnaairon** [comment]
Hey ,
I am also trying to replicate the results but I am not able too.
I followed all the steps , still I am getting varying results .
If you figured it out ,can you please help me with it ?

**Comment by janguck** [comment]
무엇이 안되시나요?

**Comment by janguck** [comment]
Hello. This is Sion Jang.

What's wrong?

If you run the command you showed, there seems to be no problem.

**Comment by eshnaairon** [comment]
Heyy , the result are very different from the one shown in the paper .

This is the best I could get , and even for this I had to increase the
number of epochs to 100 .

I want to replicate the results for the bert base model .

**Comment by janguck** [comment]
So please share your performance.

What link did you download Bert from?

**Comment by eshnaairon** [comment]
I downloaded the model from this link - https://github.com/google-research/bert (Bert-base-uncased)
I am adding a google drive link of the result - https://drive.google.com/drive/folders/10bmZTzsE8gAHZvlYMiCruviWMM0Fa32o?usp=sharing

**Comment by janguck** [comment]
Has confirmed.

With this level of performance, it seems that the author did not learn at all for reasons unknown to the author. Have you tried debugging?

If this is enough, even if you do validation on the Train dataset, you will get very low results, right?

**Comment by janguck** [comment]
Chect Metric without Training
Maybe, Seems Similar Metric.

**Comment by eshnaairon** [comment]
I have tried debugging but I am not able to figure the reason .
Yes exactly, I am getting low results .
This is the link of my code , if you can have a look - https://colab.research.google.com/drive/1GXJ3tIS7BZfbdMFYVAzl3SnwMS6KVTFC?usp=sharing

**Comment by janguck** [comment]
I entered Colab, but I am not sure because I cannot check other codes or folder structure.

However, if the performance before/after learning is similar or the same
1) When learning is not well (learning information is not reflected due to a problem in the user code)
2) After training, it is not stored well, so when Predicting, it is predicted with the BERT initial model.

I think this might be it. I hope the problem is resolved.

**Comment by eshnaairon** [comment]
I tried this but I wasn't able to figure out the problem .
Here is the directory structure ,  can you look into it?

https://drive.google.com/drive/folders/1lesPGdxpHurfHxvaZ_hTRTI9nozsP-WW?usp=sharing

**Comment by xijuan-hdu** [comment]
> I tried this but I wasn't able to figure out the problem . Here is the directory structure , can you look into it?
> 
> https://drive.google.com/drive/folders/1lesPGdxpHurfHxvaZ_hTRTI9nozsP-WW?usp=sharing

Hava you solved this problem？If you had solved this problem， could you please share the method you use. Thanks a lot！
